From 2714ae1f2eeb44ca49eef758069c925a72acebee Mon Sep 17 00:00:00 2001
From: Xiao Chen <xiao@cloudera.com>
Date: Tue, 5 Sep 2017 15:23:58 -0700
Subject: [PATCH 2631/2848] CLOUDERA-BUILD. CDH-57314. Increase the default
 number of retries in KMS clients. Addendum to fix
 logging.

Change-Id: Ic18ce76739146d6589ce3f678a489aa9697a1c93
---
 .../key/kms/LoadBalancingKMSClientProvider.java    |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/crypto/key/kms/LoadBalancingKMSClientProvider.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/crypto/key/kms/LoadBalancingKMSClientProvider.java
index 896b7a9..81b0e32 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/crypto/key/kms/LoadBalancingKMSClientProvider.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/crypto/key/kms/LoadBalancingKMSClientProvider.java
@@ -148,9 +148,9 @@ public LoadBalancingKMSClientProvider(KMSClientProvider[] providers,
               + " providers(depending on {}={} setting and numProviders={})"
               + " in the group OR the exception is not recoverable",
               CommonConfigurationKeysPublic.KMS_CLIENT_FAILOVER_MAX_RETRIES_KEY,
-              getConf().getInt(
-                  CommonConfigurationKeysPublic.
-                  KMS_CLIENT_FAILOVER_MAX_RETRIES_KEY, providers.length),
+              getConf().getInt(CommonConfigurationKeysPublic.
+                      KMS_CLIENT_FAILOVER_MAX_RETRIES_KEY,
+                  CommonConfigurationKeysPublic.KMS_CLIENT_FAILOVER_MAX_RETRIES_DEFAULT),
               providers.length);
           throw ex;
         }
-- 
1.7.9.5

